### DISCUSSION TITLE
Thanks and best wishes!

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3472] [General]
Before other contributors leave and CWA Open Source team members disband, as the Corona-Warn-App heads towards "End-of-Development", I wanted to say thank you for the teamwork! All the best to each of you as you move on to other projects or activities!

It's been a long journey of two and a half years since I submitted my first PR and a little bit longer since my first issue submission. Thanks to the Open Source team for being open to add in community suggestions and contributions!

**Comment by Ein-Tim** [comment]
Hey @MikeMcC399,

I want to give a big **Thank You!** back to you! All the time you invested in this project, the bugs you reported, the features you suggested and the PRs you provided where all highly appreciated (and seen by the project team).

Also, personally from me, a big thank you for helping me out where I was lost or had questions, carefully reviewing my PRs and being a great person to work with!

It was a pleasure working with you and I only wish you the best for your future work and projects! ❤️

**Comment by MikeMcC399** [comment]
Hi Tim,
@Ein-Tim 

Thank you very much for your nice words! I hope you are also getting some thanks from RKI for all your work as a community volunteer before you had an official position as a student intern. You, more than anybody else in the community, contributed in so many different places it was often difficult to keep track of all your improvements. Not only that, you were also helping users countless times. Well done!

All the best to you!
